### DISCUSSION TITLE
Difficulties in Citing SpaCy for a Research Paper

### DISCUSSION THREAD

**Discussion by DiamelaFrapolli** [explosion/spaCy#13683] [Help: Other Questions]
Hello everyone,

I am conducting research in linguistics where I use some natural language processing (NLP) techniques, and I want to properly cite SpaCy. I found that the most common way to cite it is as follows:

"Honnibal, M., & Montani, I. (2017). spaCy 2: Natural language understanding with Bloom embeddings, convolutional neural networks, and incremental parsing."

However, when I searched for this paper, I was unable to find it. I navigated the official SpaCy website and explored various links claiming to have the PDF, but I could only access the main SpaCy page. It's a curious situation because many people are citing a work that seems to exist, yet I cannot locate it anywhere online.

Does anyone have any information on this? Do you know if the paper was actually published or if it has been removed from the SpaCy website for some reason? If so, is there another recommended way to cite SpaCy?

I appreciate any help!

**Comment by elpeero** [comment]
Almost a year later, but I just went on the same quest as you did and couldn't find the paper either. Which shouldn't be a problem if you are just using spaCy as a tool though, because the DOI is actually referring to the repository on Zenodo now, so just cite it as software.

"Cite all versions? You can cite all versions by using the DOI [10.5281/zenodo.1212303 ](https://doi.org/10.5281/zenodo.1212303) . This DOI represents all versions, and will always resolve to the latest one."

Cite as:
Ines Montani, Matthew Honnibal, Matthew Honnibal, Adriane Boyd, Sofie Van Landeghem, & Henning Peters. (2023). explosion/spaCy: v3.7.2: Fixes for APIs and requirements (v3.7.2). Zenodo. https://doi.org/10.5281/zenodo.10009823

**Comment by sghng** [comment]
Also curious about this. Another way I can think of is cite the GitHub repo as source code

**Comment by sghng** [comment]
@DiamelaFrapolli I just found that there is a citation file in the repo: https://github.com/explosion/spaCy/blob/master/CITATION.cff.

For Zotero, i can copy the content of the file, go to Zotero -> File, and select "import from clip board"
